backend/core/src/seeds/listings/listing-detroit-10145.ts

```diff
@@ -129,6 +130,69 @@ export class Listing10145Seed extends ListingDefaultSeed {
       reservedCommunityDescription: "",
     }
 
+    const mcvUnitsSummaryToBeCreated: UnitsSummaryCreateDto[] = []
+
+    const oneBdrmUnitsSummary: UnitsSummaryCreateDto = {
+      unitType: unitTypeOneBdrm,
+      totalCount: 28,
+      monthlyRent: "$0",
```

Comment:
Yes these were the required fields.

Reply:
Yeah, weird that `monthlyRent` is required (since it's got the [`IsOptional` decorator](https://github.com/CityOfDetroit/bloom/blob/bdb1bd784c776dea08ae518a25e6ac81325aac6f/backend/core/src/units-summary/entities/units-summary.entity.ts#L23), but it must be because it also has the `PrimaryColumn` decorator). I think @abbiefarr has work in mind/progress to drop `monthlyRent` from the list of primary columns? But until then, I'd vote we set this to the empty string (as the closest way to represent the truth: we just don't know what the rent is).

Reply:
Yes, the structure of the UnitsSummary table is changing a bit [here](https://github.com/bloom-housing/bloom/pull/1670/).

Reply:
I'll start drafting up the changes based on [this](https://github.com/bloom-housing/bloom/pull/1670/). Then once its merged we can refresh with the new Listings information vs Property info.

Reply:
Sounds good! I can't tell if you're suggesting not merging this PR until https://github.com/bloom-housing/bloom/pull/1670/ is merged and then pulled down to our repo? My two cents: let's submit this with _something_ for `monthlyRent` and then come back and submit a separate PR once that other PR eventually makes its way to our repo.

Reply:
@anders-schneider SGTM